yt/visualization/plot_window.py

```diff
@@ -2466,6 +2463,13 @@ def __init__(
                 "currently supported geometries:"
                 f" {self._supported_geometries!r}"
             )
+
+        if depth is None:
+            # off-axis projection, depth not specified
+            # -> set 'large enough' depth using half the box diagonal + margin
+            depth = np.linalg.norm(ds.domain_width.in_units("code_length")) * 1.02
```

Comment:
Why 1.02?  Does it need to be an epsilon that large?

Reply:
At least for SPH data, periodicities are applied before rotation, and are never taken to be periodic in the rotated coordinates. Therefore, the large margin *shouldn't* cause trouble, but a smaller value ought to be fine as well.

Reply:
I guess I also figured that the half box diagonal would be a very large margin for most line of sight directions anyway